qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1517A.py

```diff
@@ -184,12 +551,29 @@ def _set_measurement_mode(self, mode: Union[MM.Mode, int]) -> None:
                        channels=[self.channels[0]])
                    .message)
 
+    def _set_measurement_operation_mode(self,
+                                        mode: Union[constants.CMM.Mode, int]
+                                        ) -> None:
+        self.write(MessageBuilder()
+                   .cmm(mode=mode,
+                        chnum=self.channels[0])
+                   .message)
+
+    def _get_measurement_operation_mode(self) -> list:
```

Comment:
If I understand correctly this could be `List[Tuple[str, str]]`

Reply:
Jens, thanks, it should ideally be `List[Tuple[constants.ChNr, constants.CMM.Mode]]` and `.name` calls below should not be there

Reply:
done